vignettes/articles/addnewforcer.Rmd

```diff
@@ -0,0 +1,38 @@
+---
+title: "Adding a new forcer to Hector"
```

Comment:
This article was requested by @ssmithClimate

Reply:
Thank you @kdorheim , this is great! I added a small suggestion for additional text in a comment.